src/plugins/imagery/components/ImageryView.vue

```diff
@@ -774,15 +754,41 @@ export default {
       this.stopFollowingTimeContext();
       this.timeContext = this.openmct.time.getContextForView(this.objectPath);
       //listen
-      this.timeContext.on('timeSystem', this.trackDuration);
-      this.timeContext.on('clock', this.trackDuration);
+      this.timeContext.on('timeSystem', this.timeContextChanged);
+      this.timeContext.on('clock', this.timeContextChanged);
+      this.timeContextChanged();
     },
     stopFollowingTimeContext() {
       if (this.timeContext) {
-        this.timeContext.off('timeSystem', this.trackDuration);
-        this.timeContext.off('clock', this.trackDuration);
```

Comment:
Was there a trackDuration function that can be removed?

Reply:
oh I guess we're still using it elsewhere. Why are we switching from trackDuration here?

Reply:
woah... good question?? let me look

Reply:
Ah, it's part of this fix:
https://github.com/nasa/openmct/commit/344993ae2fd96fbfbeac5df597b0a817840699c9